src/components/ChatLog.js

```diff
@@ -0,0 +1,39 @@
+import ChatEntry from '../components/ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = (props) => {
+
+  const fullChatLog = props.entries.map((entry, index) => {
+    return (
+      <li key={index}>
```

Comment:
We should only use index for the key prop when we do not have an object ID, use index as a last resort. Since you have entry.id, use that id for key instead here